pkg/local-storage/member/controller/scheduler/resources.go

```diff
@@ -239,6 +239,12 @@ func (r *resources) getAssociatedVolumes(vol *apisv1alpha1.LocalVolume) map[stri
 			r.cachePVC(pvcInCluster)
 			pvc = pvcInCluster
 		}
+
+		//Filter the bound pvc
+		if pvc.Status.Phase == "Bound" {
+			continue
+		}
+
```

Comment:
this change may cause confusion of this func `getAssociatedVolumes`, the filter logic may happen where these associated volumes are used

Reply:
This function is currently only responsible for calculating the capacity when creating a volume. I think it might be more convenient to use if we change its name.

Reply:
I modified the code.According to analysis, only the vol passed when creating vol for the first time is a virtual one without config parameters. If it is a migration or expansion scenario, it will not be affected.